draft-ietf-mls-protocol.md

```diff
@@ -648,43 +648,23 @@ For example, in the below tree:
 * The copath of C is (D, AB, EFG)
 
 ~~~~~
-              7 = root
+           ABCDEFG = root
```

Comment:
I would consider using single letters, from the end of the alphabet, for these.

Reply:
That's indeed better like this, I commited it.